compiler/luci/pass/src/ResolveCustomOpMatMulPass.cpp

```diff
@@ -121,11 +78,12 @@ bool resolve_matmul(luci::CircleCustom *cop)
   if (transpose_a)
   {
     // Create a permutation constant node
-    std::vector<uint32_t> perm;
-    for (uint32_t i = 0; i < circle_lhs->rank(); ++i)
+    std::vector<int32_t> perm;
```

Comment:
type uint32_t has changed to int32_t. looks like creating S32 requires int32_t. am I correct? but this change seems another reason for this PR. is this change necessary like compiler error?

Reply:
you've added a comment while I was reading and adding this comment.